cluster/manifests/serviceaccounts.yaml

```diff
@@ -8,8 +8,18 @@ metadata:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: jobs-operator
+  namespace: kopf
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::077346344852:role/pulse3d-pods-iam-role01
```

Comment:
I think the operator should have it's own iam policy incase we need to give it more/less permissions than what the pulse3d deployments need